Open_Acidification_pH-stat_arduino/RunAutoTune.ino

```diff
@@ -53,7 +53,7 @@ void RunAutoTune() {
       lcd.print(F("   CANCELLING   "));
       lcd.setCursor(0, 1);
       lcd.print(F("   AUTOTUNING   "));
-      delay(3000);
+      delay(3 * SECOND);
```

Comment:
Would it make sense to have a few delay length constants? It seems we use 1, 3, 5, 7 second delays.